feat: add a Figma import script

BalbinaK opened the pull request:
We can now import Figma variables by running `pnpm run import`. It will read the PROJECT_ID and FIGMA_TOKEN from `.env`.

This is what the terminal will print:
- when `FIGMA_TOKEN` is available:
<img width="432" alt="Screenshot 2024-01-30 at 14 34 52" src="https://github.com/warp-ds/figma-to-style-dictionary/assets/41303231/4ae377fa-68e7-461e-b041-98941e21587a">


- when `FIGMA_TOKEN` is missing:
<img width="597" alt="Screenshot 2024-01-30 at 14 35 25" src="https://github.com/warp-ds/figma-to-style-dictionary/assets/41303231/d991d8e8-14ec-430f-8b7e-aa4771172bc1">

BalbinaK committed:
feat: add a Figma import script

BalbinaK committed:
chore(README.md): Update the 'Download data from Figma' section

imprashast commented:
Few comments:

- Data directory can be deleted from github since it's raw data.
- Maybe we should stick to pnpm for this repo as well

imprashast reviewed (approved):
Looks good apart from 2 comments

BalbinaK committed:
Delete data directory

BalbinaK committed:
Add data/ directory to .gitignore

BalbinaK committed:
Use pnpm instead of npm

BalbinaK commented:
> * Data directory can be deleted from github since it's raw data.

Fixed in [47e9155](https://github.com/warp-ds/figma-to-style-dictionary/pull/1/commits/47e9155e516f1a3cb3a823855256bda6804025aa) and added to .gitignore in [188095d](https://github.com/warp-ds/figma-to-style-dictionary/pull/1/commits/188095d9d912f188d2e0a502b140712e1411843f)

> * Maybe we should stick to pnpm for this repo as well

Removed package-lock.json and changed `npm` mentions to `pnpm` in [62c3923](https://github.com/warp-ds/figma-to-style-dictionary/pull/1/commits/62c39231c61cc20f50f8008b51d54375ab0d008b)
